code/modules/jobs/job_types/assistant.dm

```diff
@@ -17,13 +17,22 @@ Assistant
 	paycheck = PAYCHECK_ASSISTANT // Get a job. Job reassignment changes your paycheck now. Get over it.
 	paycheck_department = ACCOUNT_CIV
 	display_order = JOB_DISPLAY_ORDER_ASSISTANT
+	var/new_player_title = "Intern"
 
 /datum/job/assistant/get_access()
 	if(CONFIG_GET(flag/assistants_have_maint_access) || !CONFIG_GET(flag/jobs_have_minimal_access)) //Config has assistant maint access set
 		. = ..()
 		. |= list(ACCESS_MAINT_TUNNELS)
 	else
 		return ..()
+/datum/job/assistant/get_title(mob/H)
+	var/final = title
+	var/client/the_client = H.client

```

Comment:
Client will only be here for late joiners, you'd need to pass down client from job equip down for roundstart ones

Reply:
god that's annoying